docs/docs/cmd/spo/file/file-sharinginfo-get.md

```diff
@@ -0,0 +1,49 @@
+# spo file sharinginfo get
+
+Generates the sharing information report for specified file
+
+## Usage
+
+```sh
+m365 spo file sharinginfo get [options]
+```
+
+## Options
+
+`-h, --help`
+: output usage information
+
+`-w, --webUrl <webUrl>`
+: The URL of the site where the file is located
+
+`-u, --url [url]`
+: The server-relative URL of the file to retrieve. Specify either `url` or `id` but not both
+
+`-i, --id [id]`
+: The UniqueId (GUID) of the file to retrieve. Specify either `url` or `id` but not both
+
+`--query [query]`
+: JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+
+`-o, --output [output]`
+: Output type. `json,text`. Default `text`
+
+`--verbose`
+: Runs command with verbose logging
+
+`--debug`
+: Runs command with debug logging
+
+## Examples
+
+Get file sharing information report for the file with server-relative url _/sites/M365CLI/Shared Documents/SharedFile.docx_ located in site _<https://contoso.sharepoint.com/sites/project-x>_
```

Comment:
That is strange. 😕 **markdownlint** is pointing it as a warning and not to use _Bare URLs_. 
![image](https://user-images.githubusercontent.com/10369964/99635202-a6996180-2a67-11eb-8be6-3085c6b58d94.png)

Guess we should not follow all the recommendation by **markdownlint**

Reply:
Right. Perhaps we should revisit it and rather than using emphasis for the typed values, use backticks to format it as code, which should prevent markdownlint complains.